src/index.spec.ts

```diff
@@ -21,9 +22,42 @@ describe("FastlyProvider", (): void => {
     fetchMock.resetMocks();
   });
 
+  describe("constructor", (): void => {
+    beforeEach((): void => {
+      fetchMock.mockResponseOnce(JSON.stringify(configFixture), {
+        headers: {
+          "access-control-allow-origin": "*",
+        },
+      });
```

Comment:
Probably the ship has sailed on this one, but perhaps you'd like to consider using MirageJS for faking the API instead of jest's fake-mock: https://miragejs.com/docs/comparison-with-other-tools/

Reply:
Yeah thats a very good suggestion and I've been meaning to use it on more projects. I'll add a ticket to do just this.